students/KellyM/Lesson3/slicing.py

```diff
@@ -0,0 +1,46 @@
+def exchange_first_last(seq):
+    """ This function takes a sequence as an argument and returns a copy of that sequence with the first and last items exchanged."""
+    first = seq[0]
+    middle = seq[1:-1]
+    last = seq[-1]
+    return last,middle,first
```

Comment:
this does not return the same data type, for example if you pass in a string - does it return a string?

you should use slicing only, `seq[0]` is indexing